src/riak_kv_overflow_queue.erl

```diff
@@ -397,7 +397,7 @@ generate_ordered_guid() ->
         [Year, Month, Day, H, M, C band 16#0fff, D band 16#3fff bor 16#8000, E]).
 
 
--spec disklog_filename(string(), string()) -> filename:filename().
+-spec disklog_filename(string(), string()) -> filename().
```

Comment:
In kv_index_tictactree you use:
`-spec disklog_filename(string(), string()) -> file:filename_all().`
The function is exactly the same (cf. aee_keystore.erl line 1245).
According to spec in filename.erl, you should use filename_all as type.